clients/src/main/resources/common/message/ShareFetchResponse.json

```diff
@@ -0,0 +1,70 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "apiKey": 78,
+  "type": "response",
+  "name": "ShareFetchResponse",
+  "validVersions": "0",
+  "flexibleVersions": "0+",
+  "fields": [
+    { "name": "ThrottleTimeMs", "type": "int32", "versions": "0+", "ignorable": true,
+      "about": "The duration in milliseconds for which the request was throttled due to a quota violation, or zero if the request did not violate any quota." },
+    { "name": "ErrorCode", "type": "int16", "versions": "0+", "ignorable": true,
+      "about": "The top level response error code." },
+    { "name": "Responses", "type": "[]ShareFetchableTopicResponse", "versions": "0+",
+      "about": "The response topics.", "fields": [
+      { "name": "TopicId", "type": "uuid", "versions": "0+", "ignorable": true,
+        "about": "The unique topic ID."},
+      { "name": "Partitions", "type": "[]PartitionData", "versions": "0+",
+        "about": "The topic partitions.", "fields": [
+        { "name": "PartitionIndex", "type": "int32", "versions": "0+",
+          "about": "The partition index." },
+        { "name": "ErrorCode", "type": "int16", "versions": "0+",
+          "about": "The error code, or 0 if there was no fetch error." },
+        { "name": "AcknowledgeErrorCode", "type": "int16", "versions": "0+",
+          "about": "The error code for acknowledgement, or 0 if there was no acknowledge error."},
+        { "name": "CurrentLeader", "type": "LeaderIdAndEpoch", "versions": "0+", "taggedVersions": "0+", "tag": 0, "fields": [
+          { "name": "LeaderId", "type": "int32", "versions": "0+",
+            "about": "The ID of the current leader or -1 if the leader is unknown." },
+          { "name": "LeaderEpoch", "type": "int32", "versions": "0+",
+            "about": "The latest known leader epoch." }
+        ]},
+        { "name": "Records", "type": "records", "versions": "0+", "nullableVersions": "0+",
+          "about": "The record data."},
+        { "name": "AcquiredRecords", "type": "[]AcquiredRecords", "versions": "0+",
+          "about": "The acquired records", "fields": [
+          { "name": "BaseOffset", "type":  "int64", "versions": "0+",
+            "about": "The earliest offset in this batch of acquired records." },
+          { "name": "LastOffset", "type": "int64", "versions": "0+", "about":
+            "The last offset of this batch of acquired records." },
+          { "name": "DeliveryCount", "type": "int16", "versions": "0+",
+            "about": "The delivery count of this batch of acquired records." }
+        ]}
+      ]}
+    ]},
+    { "name": "NodeEndpoints", "type": "[]NodeEndpoint", "versions": "0+", "taggedVersions": "0+", "tag": 0,
```

Comment:
KIP needs to correct `16+` version in both of the responses. Also do we need `taggedVersions` key?